scripts/aznfs_install.sh

```diff
@@ -75,6 +118,31 @@ use_dnf_or_yum()
     fi
 }
 
+# Custom version comparison function
+compare_versions()
+{
+    local current_version=$1
+    local latest_version=$2
+
+    # Split version strings into arrays
+    IFS='.' read -ra v1_parts <<< "$current_version"
+    IFS='.' read -ra v2_parts <<< "$latest_version"
+
+    # Compare each component of the version
+    for ((i = 0; i < ${#v1_parts[@]}; i++)); do
+        if [ "${v1_parts[i]}" -lt "${v2_parts[i]}" ]; then
+            echo "1" # current_version < latest_version
+            return
+        elif [ "${v1_parts[i]}" -gt "${v2_parts[i]}" ]; then
+            echo "-1" # current version > latest_version
+            return
+        fi
+    done
+
+    # If all components are equal, the versions are equal
+    echo "0"
+}
```

Comment:
we can simply do

is_new_version_available()
{
    local current_version_integer=$(echo $RELEASE_NUMBER | tr -d ".")
    local latest_version_integer=$(echo $1 | tr -d ".")

    [ $current_version_integer -gt $latest_version_integer ]
}

also note that we should only do what's really needed. We don't need to do a full compare (less/greater/equal), all we need to find if the given version is newer, so just do that.